.expeditor/scripts/verify/shared.sh

```diff
@@ -29,7 +29,7 @@ install_rustup() {
     echo "--- :rust: rustup is currently installed."
   else
     echo "--- :rust: Installing rustup."
-    curl https://sh.rustup.rs -sSf | sh -s -- --no-modify-path -y
+    curl https://sh.rustup.rs -sSf | sh -s -- --no-modify-path -y --profile=minimal
     # shellcheck disable=SC1090
```

Comment:
could you add this same arg to https://github.com/habitat-sh/habitat/blob/master/.expeditor/scripts/shared.ps1#L57

Reply:
Yessir!